luaui/RmlWidgets/gui_awards/gui_awards.rcss

```diff
@@ -0,0 +1,171 @@
+* {
```

Comment:
This kind of selector must be avoided at all costs. If you want to set text then you put it in the body {} selector.

What this rule is doing is putting a font property on every single element. Font family and size etc. will be inherited and set once at high level, ie the body.